swiftpipeline/config.py

```diff
@@ -155,6 +167,38 @@ def __read_config(self):
 
         with open(f"{self.config_directory}/config.yml", "r") as handle:
             self.raw_config = yaml.safe_load(handle)
+        for key in appendable_config_keys:
+            if key not in self.raw_config:
+                self.raw_config[key] = []
+
+        if "auto_plotter_registration" in self.raw_config:
+            if not isinstance(self.raw_config["auto_plotter_registration"], list):
+                self.raw_config["auto_plotter_registration"] = [
+                    self.raw_config["auto_plotter_registration"]
+                ]
+        else:
+            self.raw_config["auto_plotter_registration"] = []
```

Comment:
Since `appendable_config_keys` already contains "auto_plotter_registration", I think the above block of code can be shortened to
```suggestion
        for key in appendable_config_keys:
            if key in self.raw_config:
                if not isinstance(self.raw_config[key], list):
                    self.raw_config[key] = [self.raw_config[key]]
            else:
                self.raw_config[key] = []
```